### ISSUE TITLE
is not support react v16?

### ISSUE THREAD

**Issue by Shane0531** [Vincent-P/react-marked-markdown#63]
hello

this library have npm warning react v16

**Comment by twhite96** [comment]
Yeah. Seems to require a peer dependency of `react ^15.0.0`. Any updates on this?

**Comment by Shane0531** [comment]
i want to use on react v16

**Comment by twhite96** [comment]
@Shane0531 me too...

**Comment by MoatazGitHub** [comment]
![image](https://user-images.githubusercontent.com/17373050/42507491-a9a79b50-8445-11e8-9ed9-38eaf5b1c97d.png)
